src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costEveryDay = 40;
```

Comment:
Consider using more descriptive variable names like 'dailyCost' instead of 'costEveryDay'.